tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md

```diff
@@ -48,7 +48,6 @@ The expected failures in this file are from features in the owncloud/ocis repo.
 ### [Copy or move on an existing resource doesn't create a new version but deletes instead](https://github.com/owncloud/ocis/issues/4797)
 
 - [apiSpacesShares/moveSpaces.feature:322](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpacesShares/moveSpaces.feature#L322)
-- [apiSpacesShares/copySpaces.feature:752](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpacesShares/copySpaces.feature#L752)
```

Comment:
@aduffeck we need to take a look why the two other tests are not passing.

Reply:
Should I wait with merging this PR? Or we fix-forward?

Reply:
I'd say we fix-forward :)